.env.test

```diff
@@ -8,6 +8,7 @@ EMBEDLY_ENDPOINT=https://i.embed.ly.test/1/display
 GALAXY_API_BASE=https://galaxy-staging-herokuapp.com
 GEMINI_API_BASE=https://media.artsy.net
 GEMINI_ENDPOINT=https://gemini.cloudfront.test
+GEODATA_API_BASE=http://artsy-geodata.s3-website-us-east-1.amazonaws.com
```

Comment:
I left this breadcrumb, now that this data is being leaned on more:
https://artsy-geodata.s3.amazonaws.com/partner-cities/README.txt

Reply:
Thanks!